component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java

```diff
@@ -515,34 +515,26 @@ private void validateAndGetRequiredParameters() {
         emailProperties.put(EmailConstants.TRANSPORT_MAIL_HEADER_CONTENT_TYPE, contentType);
     }
 
-    @Override public void disconnect() {
+    @Override public void disconnect() {}
```

Comment:
As per wso2 coding standards, we need to format empty methods as below:
@Override public void disconnect() {
}

Could you please fix this?